client/tx/legacy_test.go

```diff
@@ -59,7 +59,7 @@ func buildTestTx(t *testing.T, builder client.TxBuilder) {
 
 type TestSuite struct {
 	suite.Suite
-	codec    codec.Codec
+	codec    codec.Codec //nolint:unused
```

Comment:
In test, if a field is unused, I'd rather remove it (for structs) or check it (for errors)

Reply:
will do :)